images/virtualization-artifact/pkg/controller/vm/internal/block_device.go

```diff
@@ -301,7 +302,14 @@ func (h *BlockDeviceHandler) countReadyBlockDevices(vm *virtv2.VirtualMachine, s
 			}
 
 			if vd.Status.Phase == virtv2.DiskReady {
-				ready++
+				inUseCondition, _ := service.GetCondition(vdcondition.InUseType, vd.Status.Conditions)
+				if inUseCondition.Status == metav1.ConditionFalse {
+					ready++
+				} else {
+					msg := fmt.Sprintf("virtual disk %s is using for create virtual image or cluster virtual image", vd.Name)
```

Comment:
The virtual disk %s is being used to create a virtual image or a cluster virtual image.

Reply:
It may not be used by vi/cvi, but may be Unknown as well

Reply:
Fixed.